packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -35,11 +35,19 @@ export default class GenericProvider {
     return true;
   }
 
-  async screenshot(name) {
+  async screenshot(name, {
+    ignoreRegionXpaths = [],
+    ignoreRegionSelectors = [],
+    ignoreRegionSeleniumElements = [],
+    customIgnoreRegions = []
```

Comment:
while you are giving defaults here, they are usually not useful. Ideally add defaults in first lines of function. The reason being if i call above with 
`screenshot("abc", { ignoreRegionXpaths: ['some', 'paths'] })`
in the function `ignoreRegionSelectors` will be `undefined` and not `[]` as you have given in defaults.